lib/main_publiccloud.pm

```diff
@@ -0,0 +1,118 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_publiccloud;
+use Mojo::Base 'Exporter';
+use utils;
+use version_utils;
+use main_common qw(loadtest load_extra_tests_prepare);
+use testapi qw(check_var get_var);
+
+our @EXPORT = qw(
+  load_publiccloud_tests
+);
+
+sub load_podman_tests() {
+    loadtest 'containers/podman';
+    loadtest 'containers/podman_3rd_party_images';
+}
+
+sub load_docker_tests() {
```

Comment:
Are we running any ARM machine in PC? If so, we should disable some of these tests for aarch64, as some packages are not available in certain versions. 
See https://confluence.suse.com/display/ENGCTNRSTORY/Level+Test+Plan#LevelTestPlan-PackagesfromContainersModule

Reply:
yes we do . for AWS we have aarch64 flavors

Reply:
i just removed `docker_image`. This doesnt be used anywhere. the rest of the modules looks ok. VRs is against ARM btw

Reply:
*this change is ready for review*

Reply:
`zypper_docker` and `runc` shouldn't be run against  <=SLE 15 in aarch64.
`registry`  shouldn't be run against  <=SLE 15-SP1 in aarch64.

Reply:
*this change is ready for review* i added conditions. thanks @jlausuch

Reply:
thanks